app/lib/shared/search_client.dart

```diff
@@ -50,6 +51,19 @@ class SearchClient {
     return result;
   }
 
+  /// Search service maintains a separate index in each of the running instances.
+  /// At the moment we cannot guarantee that each of them will receive the
+  /// notification.
+  Future triggerReindex(String package) {
+    // We have a good chance that Appengine's round-robin will send the requests
+    // to separate service instances.
+    final List<Future> requests = new List.generate(
+        10,
```

Comment:
:-/  Please add a TODO here. It is possible to query the admin api to get the list of instances.

Reply:
While the only caller is the tool that sends notifications manually, I think this is not an urgent issue. I'll add a TODO in a follow-up.